.env.dev

```diff
@@ -32,13 +32,17 @@ STAKING_POOL_FACTORY_ADDRESS=0x4b2A127680320eD980beAa7aD9b2447B96BC32fC
 RESOLVER_V1_ADDRESS=0xf5EA22c1F799d425356c2aab2004200Ab4490D2b
 CHAIN_REQUEST_ATTEMPTS=5
 
-# ASSETS 
+# ASSETS
 ASSETS_MANAGER_ADDRESS=0xE258fA7D1cc8964D0dEB7204Df947bCa42b2c940
 ASSETS_SYNC_INTERVAL_IN_HOURS=10
 ASSETS_SYNC_HISTORY_INTERVAL_IN_HOURS=21
 ASSETS_SYNC_ENABLED=true
 
 # IPFS
+IPFS_CLIENT_HOST=
+IPFS_CLIENT_PORT=
+IPFS_CLIENT_PROJECT_ID=
+IPFS_CLIENT_PROJECT_SECRET=
 
 # INTERVALS
 DIDDOC_SYNC_INTERVAL_IN_HOURS=1
```

Comment:
Do you think that local ssi-hub should be synchronized with dev? If so, then we should probably trigger synchronization immediately after ssi-hub is launched. 
May be it is enaugh to cache only DID used for login

Reply:
I am not sure. @jrhender , what is your opinion?

Reply:
Yeah, IMO caching only the DID used for login / testing would be enough.